distribution/test/itests/test-itests-alliance/src/test/java/alliance/test/itests/TestNsili.java

```diff
@@ -115,49 +119,73 @@ public void beforeAllianceTest() throws Exception {
         }
     }
 
+    /**
+     * Determine if the Nsili Source has been configured in Alliance and available
+     *
+     * @throws Exception
+     */
     @Test
     public void testNsiliHttpSourceAvailable() throws Exception {
-        // Determine if the Nsili Source has been configured in Alliance and available
         // @formatter:off
         given().auth().basic("admin", "admin").when().get(ADMIN_ALL_SOURCES_PATH.getUrl()).then()
                 .log().all().assertThat().body(containsString("\"id\":\"httpNsiliSource\""));
         // @formatter:on
     }
 
+    /**
+     * Determine if the Nsili Source has been configured in Alliance and available
```

Comment:
We may want to specify that this Nsili Source is FTP, since two are configured in the itests